crates/sui-indexer-alt-restorer/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod archives;
+mod snapshot;
+
+use archives::ArchivalCheckpointInfo;
+use clap::Parser;
+use sui_pg_db::DbArgs;
+
+use crate::snapshot::SnapshotRestorer;
+
+#[derive(Parser, Debug, Clone)]
+#[clap(name = "sui-indexer-restorer")]
```

Comment:
nit: `-alt-`? (I think this is the default if you don't specify a `clap(name = ...)`